### DISCUSSION TITLE
2-sex models in population dynamics

### DISCUSSION THREAD

**Discussion by kellijohnson-NOAA** [ss3sim/ss3sim#318] [Q&A]
## Question

I am wondering why we even have 1-sex models given that we can mirror all of the male parameters to be the same as the female parameters? 

### Issue that started this discussion

Recently, @ChristineStawitz-NOAA posted an [issue](#311) about working with two-sex models that notes 2 sexes is currently not working in ss3sim. 

### Random

I am in the process of building a converter that will take a single-sex model and add the necessary parameter lines to make it a two-sex model and increase the number of columns of the composition data.

I asked @Rick-Methot-NOAA if the estimation of parameters would be expected to be any different if they were estimated independently or using offsets and he said the end result should be the same.

**Comment by Rick-Methot-NOAA** [comment]
Only downside I can think of is that nearly twice as many calculations need to be done if there are 2 sexes rather than 1, so model will be slower.

**Reply by kellijohnson-NOAA** [reply]
Right, thanks @Rick-Methot-NOAA I didn't think about model run time or file size. I should test out the hake model in both MCMC and MLE modes to see how much longer it would take if we were to move it to a two-sex model.
